src/main/java/xyz/nucleoid/bedwars/game/active/shop/BwItemShop.java

```diff
@@ -49,7 +49,7 @@ private static void addNavbar(ShopBuilder shop, ServerPlayerEntity player, BwAct
                 player.openHandledScreen(BwItemShop.createBlocks(player, game));
             }));
         }
-        if(pageIndex==2){
+        if(pageIndex==2) {
```

Comment:
formatting nitpick: can it rather be `if (pageIndex == 2) {`? note the spaces. maybe run the intellij formatter over the code?